web/cobrands/buckinghamshire/assets.js

```diff
@@ -17,7 +17,7 @@ var defaults = {
     format_class: OpenLayers.Format.GML.v3.MultiCurveFix,
     asset_type: 'spot',
     max_resolution: 4.777314267158508,
```

Comment:
See e.g. something like 88f8334157416f33d0bd81a38b46b25005cbbc15 for if we don't want the assets to appear wildly more zoomed out than normal on the Bucks cobrand?